Gemfile

```diff
@@ -17,11 +17,14 @@ group :development do
   gem 'rspec'
   gem 'shotgun'
   gem 'solargraph'
+  gem 'debase'
+  gem 'ruby-debug-ide', require: false
```

Comment:
I have yet to really understand ruby's whole dependency and bundler ecosystem, but why isn't `debase` also `require: false`? does our actual code call it anywhere? shouldn't it be not required like ruby-debug-ide

Reply:
Didn't include it because that's not how they said it in their readme. I think it needs to not be there